<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.9.4" xml:lang="en-US">
  <compounddef id="classmodules_1_1exceptions_1_1ReturnError" kind="class" language="Python" prot="public">
    <compoundname>modules::exceptions::ReturnError</compoundname>
    <basecompoundref refid="classmodules_1_1exceptions_1_1Error" prot="public" virt="non-virtual">modules.exceptions.Error</basecompoundref>
      <sectiondef kind="public-attrib">
      <memberdef kind="variable" id="classmodules_1_1exceptions_1_1ReturnError_1ae8bc12e8c4250a64ba9434418476cb96" prot="public" static="no" mutable="no">
        <type></type>
        <definition>modules.exceptions.ReturnError::msg</definition>
        <argsstring></argsstring>
        <name>msg</name>
        <qualifiedname>modules.exceptions.ReturnError.msg</qualifiedname>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="Backends/scripts/BOSS/modules/exceptions.py" line="21" column="1" bodyfile="Backends/scripts/BOSS/modules/exceptions.py" bodystart="21" bodyend="-1"/>
      </memberdef>
      </sectiondef>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="classmodules_1_1exceptions_1_1ReturnError_1a52bd3a7f43343d20a110bddb946396bd" prot="public" static="no" const="no" explicit="no" inline="no" virt="non-virtual">
        <type>def</type>
        <definition>def modules.exceptions.ReturnError.__init__</definition>
        <argsstring>(self, msg)</argsstring>
        <name>__init__</name>
        <qualifiedname>modules.exceptions.ReturnError.__init__</qualifiedname>
        <param>
          <type>self</type>
          <defname>self</defname>
        </param>
        <param>
          <type>msg</type>
          <defname>msg</defname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="Backends/scripts/BOSS/modules/exceptions.py" line="20" column="1" bodyfile="Backends/scripts/BOSS/modules/exceptions.py" bodystart="20" bodyend="22"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
<para><verbatim>Exception raised when a function cannot return the expected result.

Attributes:
    msg  -- explanation of the error
</verbatim> </para>
    </detaileddescription>
    <inheritancegraph>
      <node id="3">
        <label>Exception</label>
      </node>
      <node id="1">
        <label>modules.exceptions.ReturnError</label>
        <link refid="classmodules_1_1exceptions_1_1ReturnError"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
      </node>
      <node id="2">
        <label>modules.exceptions.Error</label>
        <link refid="classmodules_1_1exceptions_1_1Error"/>
        <childnode refid="3" relation="public-inheritance">
        </childnode>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="3">
        <label>Exception</label>
      </node>
      <node id="1">
        <label>modules.exceptions.ReturnError</label>
        <link refid="classmodules_1_1exceptions_1_1ReturnError"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
      </node>
      <node id="2">
        <label>modules.exceptions.Error</label>
        <link refid="classmodules_1_1exceptions_1_1Error"/>
        <childnode refid="3" relation="public-inheritance">
        </childnode>
      </node>
    </collaborationgraph>
    <location file="Backends/scripts/BOSS/modules/exceptions.py" line="13" column="1" bodyfile="Backends/scripts/BOSS/modules/exceptions.py" bodystart="13" bodyend="22"/>
    <listofallmembers>
      <member refid="classmodules_1_1exceptions_1_1ReturnError_1a52bd3a7f43343d20a110bddb946396bd" prot="public" virt="non-virtual"><scope>modules::exceptions::ReturnError</scope><name>__init__</name></member>
      <member refid="classmodules_1_1exceptions_1_1ReturnError_1ae8bc12e8c4250a64ba9434418476cb96" prot="public" virt="non-virtual"><scope>modules::exceptions::ReturnError</scope><name>msg</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
